pkg/service/main.go

```diff
@@ -25,17 +25,17 @@ import (
 	"github.com/gorilla/mux"
 )
 
-var serviceNameOverride string
+var instanceName string
 
 func Main(serviceName string, serviceVersion string, proto interface{}, ctx context.Context, cancel context.CancelFunc, router *mux.Router, readyStream chan<- bool) {
 	startupTimer := startup.NewStartUpTimer(common.BootRetrySecondsDefault, common.BootTimeoutSecondsDefault)
 
 	additionalUsage :=
-		"    -n, --serviceName               Overrides the service name to be stored in metadata, used with Registry and/or Configuration Providers\n" +
-			"                                    If the profile is also provided, name will be replaced with \"<name>;<profile>\"\n"
+		"    -i, --instance                  Provides a service name suffix which allows unique instance to be created\n" +
+			"                                    If the option is provided, service name will be replaced with \"<instance>_<name>\"\n"
```

Comment:
the instanceName is suffix, not prefix, so it should be <name >+ "_" + <instance>